proto/acl.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package acl;
+
+// Access control list for BuildBuddy objects.
+message ACL {
+  Permissions owner_permissions = 1;
+  Permissions group_permissions = 2;
+  Permissions others_permissions = 3;
+
+  message Permissions {
+    bool read = 1;
+    bool write = 2;
+    bool execute = 3;
```

Comment:
let's just remove this for now since we're not using it. Don't want clients to rely on it until we know how we're using it.

Reply:
Done